lms/djangoapps/courseware/views.py

```diff
@@ -651,6 +652,11 @@ def course_info(request, course_id):
     course_key = SlashSeparatedCourseKey.from_deprecated_string(course_id)
     with modulestore().bulk_operations(course_key):
         course = get_course_by_id(course_key, depth=2)
+
+        if not has_access(request.user,
+           'view_courseware_with_prerequisites', course) and isinstance(course_key, CCXLocator):
```

Comment:
What's this user permission, `view_courseware_with_prerequisites` ?

Does a staff user typically have that permission on a course?